ACMAS/Docker deployment instructions.md

```diff
@@ -0,0 +1,20 @@
+# Deployment instructions for ACMAS @ RCOS v2 (Django, Docker, Postgres, Gunicorn, Nginx)
+## Production server
+_Note: Running migrations and serving static files are now handled in entrypoint.prod.sh. and entrypoint.sh._
+
+[Docker](https://docs.docker.com/get-docker/) is a pre-requisite installation to run the deployment server.
+If you do not have Docker installed, please do so before continuing to the remainder of the instructions.
+### Build images and spin up production containers
+`docker-compose -f docker-compose.prod.yml up -d --build`
```

Comment:
make sure to make a comment that the user needs the additional .env.prod and .env.prod.db files in the ACMAS-Frontend/ACMAS folder for the command to run. Also specify that this command line needs to be run in the command line in the ACMAS-Frontend/ACMAS directory

Reply:
make sure to specify the crlf to lf error